psi4/driver/procrouting/solvent/pol_embed.py

```diff
@@ -0,0 +1,120 @@
+#
+# @BEGIN LICENSE
+#
+# Psi4: an open-source quantum chemistry software package
+#
+# Copyright (c) 2007-2019 The Psi4 Developers.
+#
+# The copyrights for code used from other parties are included in
+# the corresponding files.
+#
+# This file is part of Psi4.
+#
+# Psi4 is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation, version 3.
+#
+# Psi4 is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License along
+# with Psi4; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+# @END LICENSE
+#
+
+import numpy as np
+import cppe
+
+from psi4 import core
+
+
+def psi4mol_to_cppemol(psi4mol):
+    mol = cppe.Molecule()
+    geom = psi4mol.geometry().np
+    for i, c in enumerate(geom):
+        a = cppe.Atom((int(psi4mol.Z(i))), *c)
```

Comment:
be good to try this on something with a ghost atom (`Z(i)` will be 0.0) to make sure nothing breaks and result is sane.

Reply:
Result looks good with a ghost atom.